htsget-http-actix/src/main.rs

```diff
@@ -1,59 +1,21 @@
 use std::env::args;
 use std::sync::Arc;
 
-use crate::handlers::blocking::{get, post, reads_service_info, variants_service_info};
+//use htsget_http_actix::handlers::blocking::{get, post, reads_service_info, variants_service_info};
 use actix_web::{web, App, HttpServer};
-use config::Config;
-use handlers::{
+use htsget_http_actix::handlers::{
```

Comment:
This will probably need a `#[cfg(feature = "async")]`, because the use won't work unless the `async` feature is active.

Together with the other change, this means we don't need the aliases, if I am not wrong

Reply:
Makes sense, went that way for now, but there's a cascade of changes needed after that, let's see if we can wrap it up well...